src/schema/community-board-budget-request.ts

```diff
@@ -49,3 +62,62 @@ export const cbbrOptionCascade = pgTable(
     ),
   ],
 );
+
+export const cbbrAgencyCategoryResponse = pgTable(
+  "cbbr_agency_category_response",
+  {
+    id: smallint("id").generatedByDefaultAsIdentity().primaryKey(),
+    description: text("description").notNull(),
+  },
+);
+export const communityBoardBudgetRequest = pgTable(
+  "community_board_budget_request",
+  {
+    id: text("id").notNull().primaryKey(),
+    trackingCode: text("tracking_code").notNull(),
+    boroughId: char("borough_id", { length: 1 })
+      .notNull()
+      .references(() => borough.id),
+    communityDistrictId: char("community_district_id", { length: 2 }).notNull(),
+    agency: text("agency")
+      .notNull()
+      .references(() => agency.initials),
+    managingCode: char("managing_code", { length: 3 })
+      .notNull()
+      .references(() => managingCode.id),
+    agencyCategoryResponse: smallint("agency_category_response_id")
+      .notNull()
+      .references(() => cbbrAgencyCategoryResponse.id),
+    agencyResponse: text("agency_response"),
+    requestType: text("request_type").notNull(),
+    priority: smallint("priority").notNull(),
+    need: smallint("need_id")
+      .notNull()
+      .references(() => cbbrNeed.id),
+    request: smallint("request_id").references(() => cbbrRequest.id),
+    explanation: text("explanation"),
+    isLocationSpecific: boolean("is_location_specific").notNull(),
+    liFtMPnt: multiPointGeom("li_ft_m_pnt", 2263),
+    liFtMPoly: multiPolygonGeom("li_ft_m_poly", 2263),
+    mercatorLabel: pointGeom("mercator_label", 3857),
+    mercatorFillMPnt: multiPointGeom("mercator_fill_m_pnt", 3857),
+    mercatorFillMPoly: multiPolygonGeom("mercator_fill_m_poly", 3857),
+  },
+  (table) => [
+    foreignKey({
+      columns: [table.boroughId, table.communityDistrictId],
+      foreignColumns: [communityDistrict.boroughId, communityDistrict.id],
+    }),
+    check(
+      "community_board_budget_request_request_type_options",
+      sql`${table.requestType} IN (
+        'Capital',
+        'Expense'
+      )`,
+    ),
+    index().using("GIST", table.liFtMPnt),
```

Comment:
The API will be looking up budget requests based on the tracking code. It might be worth considering adding an index to that column, as well